config/locales/en.yml

```diff
@@ -85,6 +85,10 @@ en:
       show_dashboard: Show dashboard
       reset: Reset
       save_key: "Note that we won't be able to show the key to you again. New API key:"
+      multifactor_auth: Multifactor authentication
```

Comment:
Both `multifactor_auth` and `enable_mfa` are referenced in `edit.html` and `new.html`, but not in `index.html`. Can we remove thoses keys from `index:` and instead declare them in `edit:` and `new:`?

Also, `mfa_api_enabled` is only used in `new.html`.  Better declare the key under `new:` instead.